course-preparation/Ansible/training-lab.yml

```diff
@@ -14,69 +14,32 @@
     include_vars:
       file: "group_vars/passwords-{{ site_name }}.yml"
 
-  - name: Announce the setup
```

Comment:
Please keep the task in here, but I'll add a tag (for slack), and update the web hook so that we don't have the 404.